packages/@dcl/inspector/src/components/Toolbar/Toolbar.tsx

```diff
@@ -27,8 +31,15 @@ const Toolbar = withSdk(({ sdk }) => {
     []
   )
 
+  const handleSave = useCallback(async () => {
+    await sdk.dataLayer.save({})
```

Comment:
what do you think about adding the `save` to the `useSave` hook, so if needed somewhere else it can be retrieved from the `useSave` hook too?

Reply:
good idea